config/samples/dataplane_v1beta1_openstackdataplanenodeset_sriov.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-sriov
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+    - name: ANSIBLE_ENABLE_TASK_DEBUGGER
+      value: "True"
+  services:
+    - download-cache
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - neutron-sriov-agent
```

Comment:
I'd suggest we use a more generic name (neutron-sriov instead of neutron-sriov-agent) in case / when we have to do something else but only the agent deployment as part of this Service.

Reply:
Done